### ISSUE TITLE
[Admissions Redesign - Core] Epic - Counselors Page/Functionality

### ISSUE THREAD

**Issue by mjoneill** [uiowa/uiowa#2212]
Unique fields specific the the "Counselor" person type along with several different displays.

Counselors Landing Page – https://admissions.uiowa.edu/counselors
Counselors Teritory Page - https://admissions.uiowa.edu/counselor/IA
Example Counselor Profile - https://admissions.uiowa.edu/counselor/jessica-klein

# Stories

- ~~Extend person content type to hold counselor information #2484~~
- ~~Investigate Mapping Solutions in Drupal 8/9 #2499~~
- ~~Counselor Map #2485~~
- ~~Counselor Listing Display #2486~~
- Attach person to local event #2487

# Original Issue Summary
<details>
* map or way to find your counselor (geographically)?
* Iowa Update counselor landing page? News items/blog for counselors?

>They would like the ability to search for counselors by state (map) and having the ability to view all counselors. They currently have a select list along with the map to search by state for an accessible alternative to the map. When using these options it takes you to a teaser view of the counselors for that state and allows you to click their name to see everything about them.

>They are fine with us taking an initial stab at the design of this and providing feedback once they have something to react to.

>There can be multiple counselors per state as well as a single counselor can cover multiple states.

Name - currently using title field - needs to be a text field
Job Title - currently text field
Email - email field
Degree(s) - text field
Hometown - text field
What would people be surprised to know about you? - long text - no WYSIWYG
My favorite thing about working in Admissions - long text - no WYSIWYG
Iowa City “Must Do and Must See” Locations - long text - no WYSIWYG??
What makes Iowa such a great place? - long text - no WYSIWYG
Territory - select list of states but does have "international recruitment" as an option. multiple
Profile Image - image with focal point
</details>

**Comment by mjoneill** [comment]
https://admissions.uiowa.edu/counselors - The map...

**Comment by briand44** [comment]
They would like the ability to search for counselors by state (map) and having the ability to view all counselors. They currently have a select list along with the map to search by state for an accessible alternative to the map. When using these options it takes you to a teaser view of the counselors for that state and allows you to click their name to see everything about them.

They are fine with us taking an initial stab at the design of this and providing feedback once they have something to react to.

**Comment by briand44** [comment]
Structure: There may need to be a conversation on whether this should be a separate content type of if we customize the person content type to work for counselors.

There can be multiple counselors per state as well as a single counselor can cover multiple states.

- Name - currently using title field - needs to be a text field
- Job Title - currently text field
- Email - email field
- Degree(s) - text field
- Hometown - text field
- What would people be surprised to know about you? - long text - no WYSIWYG
- My favorite thing about working in Admissions - long text - no WYSIWYG
- Iowa City “Must Do and Must See” Locations - long text - no WYSIWYG??
- What makes Iowa such a great place? - long text - no WYSIWYG
- Territory - select list of states but does have "international recruitment" as an option.
- Profile Image - image with focal point

**Comment by briand44** [comment]
The only other thing worth mentioning is that they would like to be able to relate a counselor to some events. I didn't find an example of this on their existing site but maybe @scottfi can share an example.

**Comment by scottfi** [comment]
Event with Counselor connected/listed: https://admissions.uiowa.edu/calendar/rockhurst-high-school-college-fair
(this isn't currently published on the site since it is from a while back so you'll need to be logged in to see it)

**Comment by joewhitsitt** [comment]
No longer blocked by https://github.com/uiowa/uiowa/issues/2317. Extending the person content via #2291  is the current approach.

**Comment by joewhitsitt** [comment]
Blocked by #2291

**Comment by joewhitsitt** [comment]
This is the last issue related to the https://github.com/uiowa/uiowa/issues/2212 Counselors epic

**Comment by briand44** [comment]
Closing this epic since there is a separate issue for that last remaining item.
